### ISSUE TITLE
Our Ip address is blocked

### ISSUE THREAD

**Issue by tomhughes** [tomhughes/trac-tickets#5265]
**[Submitted to the original trac issue database at 2.36pm, Tuesday, 13th January 2015]**

IP address 37.153.228.170 appears blocked for tile servers.

Kindly unblock us so we can work again :-)

This is the address of the place we program our websites from (mostly non-profit work for putting nature information on maps and in spatial db's). We were not aware that we put an unfair load on any servers, feel free to contact me if we did anything wrong.

**Comment by tomhughes** [comment]
**Author: TomH**
**[Added to the original trac issue at 2.54pm, Tuesday, 13th January 2015]**

This tracker is for issues with the web site, so operational issues with the tile servers are out of scope here.

That said, it would be very unusual to be blocked from the tile servers, as what normally happens is simply that a slow down is applied to addresses that are overusing them. Normal web usage would not generally trigger that however.

It's probably best if you can join #osm-dev on the OFTC IRC network and we'll be able to work through the problem with you. If you can't do that then please contact support@openstreetmap.org with details of the symptoms you are seeing.

**Comment by tomhughes** [comment]
**Author: TomH**
**[Added to the original trac issue at 2.55pm, Tuesday, 13th January 2015]**

Oh, you can also use http://irc.openstreetmap.org/ to join IRC and talk to us.
